core/src/test/resources/sql/measure.iq

```diff
@@ -26,6 +26,136 @@ from emp;
 
 !update
 
+create view empm_comm as
```

Comment:
I wouldn't create a view for just one query. (Unless a view is necessary to demonstrate the behavior, and a CTE wouldn't do it.)

If it's not too disruptive of later tests, you could add `avg_comm` to `empm`.